core/model/collection.py

```diff
@@ -300,26 +324,30 @@ def default_loan_period(self, library, medium=EditionConstants.BOOK_MEDIUM):
         collection has it for this number of days.
         """
         return (
-            self.default_loan_period_setting(library, medium).int_value
+            self.default_loan_period_setting(library, medium)
             or self.STANDARD_DEFAULT_LOAN_PERIOD
         )
 
-    def default_loan_period_setting(self, library, medium=EditionConstants.BOOK_MEDIUM):
+    def default_loan_period_setting(
+        self, library, medium=EditionConstants.BOOK_MEDIUM, set_value=None
```

Comment:
Since the earlier implementation was returning the configurationsettings object which allowed setting of the value, the additional `set_value` argument allows us to do the same.